docs/kernel-versions.md

```diff
@@ -147,6 +154,7 @@ Helper | Kernel version | Commit
 `BPF_FUNC_skb_under_cgroup()` | 4.8 | [4a482f34afcc](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=4a482f34afcc162d8456f449b137ec2a95be60d8)
 `BPF_FUNC_skb_vlan_pop()` | 4.3 | [4e10df9a60d9](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=4e10df9a60d96ced321dd2af71da558c6b750078)
 `BPF_FUNC_skb_vlan_push()` | 4.3 | [4e10df9a60d9](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=4e10df9a60d96ced321dd2af71da558c6b750078)
+`BPF_FUNC_sock_map_update()` | 4.14 | [174a79ff9515](https://git.kernel.org/cgit/linux/kernel/git/davem/net-next.git/commit/?id=174a79ff9515f400b9a6115643dafd62a635b7e6)
```

Comment:
This link is for BPF_FUNC_sk_redirect_map.

Reply:
Yes it is, but both helpers were added in the same commit, along with the `BPF_MAP_TYPE_SOCKMAP` map type. Even though the commit log does not mention `BPF_FUNC_sock_map_update()`.